sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -238,7 +242,7 @@ export interface ListSnapshotsOptions extends OperationOptions, ListSnapshots, O
 }
 
 // @public
-export interface ListSnapshotsPage extends HttpResponseField<SyncTokenHeaderField>, PageSettings {
+export interface ListSnapshotsPage extends SyncTokenHeaderField, PageSettings {
```

Comment:
I was thinking about either return an array for page, similar to keyvault 
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-keys/src/index.ts#L1088

or if we really need continuation token, extending the array, like metrics advisor does

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorAdministrationClient.ts#L366

So we don't need to wrap the result with `{ items }`.  Thoughts?

Reply:
@witemple-msft didn't you figure out that attaching extra properties to an array disabled a lot of runtime optimizations?

Reply:
I think that was only for configuring the property descriptors of integer props of the array.

Reply:
That sounds good, but will that be inconsistent with the `listConfigurationSettings`?

Reply:
> That sounds good, but will that be inconsistent with the `listConfigurationSettings`?

Right :(,  I am fine with keeping consistency within this package.